contracts/FlowIDTableStaking.cdc

```diff
@@ -1822,6 +1823,67 @@ access(all) contract FlowIDTableStaking {
         return true
     }
 
+    /// Validates that a networking address is properly formatted
+    /// Requirements:
+    /// 1. Must not be an IP address
+    /// 2. Must contain a port number after a colon
+    /// 3. Must be a valid domain name format
+    access(contract) view fun isValidNetworkingAddress(address: String): Bool {
+        // Split the address into domain and port
+        let parts = address.split(separator: ":")
+        if parts.length != 2 {
+            return false
+        }
+
+        let domain = parts[0]
+        let port = parts[1]
+
+        // Check if port is a valid number between 1 and 65535
+        let portNum = UInt16.fromString(port)
+        if portNum == nil || portNum! < 1 || portNum! > 65535 {
+            return false
+        }
+
+        // Check if domain is an IP address (contains only numbers and dots)
+        let domainParts = domain.split(separator: ".")
+        if domainParts.length == 4 {
+            var isIP = true
+            for part in domainParts {
+                let num = UInt8.fromString(part)
+                if num == nil || num! > 255 {
+                    isIP = false
+                    break
+                }
+            }
+            if isIP {
+                return false
+            }
+        }
+
+        // Check if domain has at least one dot and valid characters
+        if !domain.contains(".") {
+            return false
+        }
```

Comment:
here better way is to check last segment is not numeric.

Reply:
excellent suggestion!

Reply:
I have update the changes to be more robust.